### PR TITLE
Jb47487

### PR DESCRIPTION
**Pull request by vigejolla**

### CODE REVIEW

File: rpm/sdk-setup.spec [hunk 1]
```diff
@@ -225,6 +226,10 @@ chmod 755 %{buildroot}%{_bindir}/*
 
 %pre -n sdk-chroot
 if ! rpm --quiet -q ca-certificates && [ -d /%{_sysconfdir}/ssl/certs ] ; then echo "Cleaning up copied ssl certs. ca-certificates should now install"; rm -rf /%{_sysconfdir}/ssl/certs ;fi
+rm -Rf /home/.zypp-cache
```
Comment (abranson) [inline rpm/sdk-setup.spec]: If we decide later to clean this up where it was created, it might go away. Should this check for existence before removing?

**Comment by vigejolla** [comment]
> I think it might be better to check for the existence of /home/.zypp-cache before removing it, as it might later get cleaned up by whatever creates it. But it's not important right now.

That's why there is the -f flag in the rm command

**Comment by martyone** [comment]
> I would consider filing a separate bug for the zypp-cache fix.

...so that it can be linked to the bug Andrew created today

**Comment by vigejolla** [comment]
> > I would consider filing a separate bug for the zypp-cache fix.
> 
> ...so that it can be linked to the bug Andrew created today

Yeah, I guess that's best. I'll keep the fix in this PR though, I'll just add a mention to the new bug to the commit message

**Pull request merged**